### PR TITLE
Adjustment to January 2020 CBO economic projections due to COVID-19

### PR DESCRIPTION
**Pull request by chusloj**
This PR updates taxdata to reflect updates to CBO Economic Projects due to COVID-19. The general steps I took were:

1. Follow instructions in [CBO_Baseline_Updating_Instructions.md](https://github.com/PSLmodels/taxdata/blob/master/puf_stage1/doc/CBO_Baseline_Updating_Instructions.md)
2. Run `make all`

~~Updated data is pulled from the CBO's May 2020 projections for 2020-2021~~

**UPDATE**: New data is now pulled from the CBO's [July 2020 Economic Projections for 2020-2030](https://www.cbo.gov/publication/56442). All variables listed under [**Variables in 10-Year CBO Projections**](https://github.com/PSLmodels/taxdata/blob/master/puf_stage1/doc/CBO_Baseline_Updating_Instructions.md#variables-in-10-year-cbo-projections) within `CBO_Baseline_Updating_Instructions.md` were updated and unemployment compensation (**UCOMP**) was updated with March 2020 data (see [this](https://github.com/PSLmodels/taxdata/pull/332#issuecomment-654432704) comment).

### CODE REVIEW

**Comment by MattHJensen** [comment]
Thank you for opening this PR @chusloj! Looking forward to following your progress.

**Comment by chusloj** [comment]
**UPDATE**: `make all` has been run

The 2019 values for all variables listed under [**Variables in 10-Year CBO Projections**](https://github.com/PSLmodels/taxdata/blob/master/puf_stage1/doc/CBO_Baseline_Updating_Instructions.md#variables-in-10-year-cbo-projections) within `CBO_Baseline_Updating_Instructions.md` were updated to reflect actual (not projected) values.

After running the makefile, `test_growfactors.py` shows that the 2022 growth rate within `growfactors.csv` for `ADIVS` (Dividend Income, Personal) exceeds the growth rate limit of 1.6 set in `test_growfactors.py` by 6%. Would like @MattHJensen to comment on this.

Minor updates have been made to the StatMatch README and general README file. Documentation for the `cps-matched-puf.csv` is currently being constructed.

**Comment by chusloj** [comment]
The maximum grow factor within `test_growfactors.py` has been increased from 1.6 to 1.7 @MattHJensen

**Comment by andersonfrailey** [comment]
Thanks for working on this, @chusloj! With the merging of #314, you'll need to recreate the CPS weights. I haven't updated documentation for this yet, but if you just run `cps_stage2/stage2.py` after pulling all the updates to your local branch you should be good to go.

**Comment by chusloj** [comment]
**UPDATE**: The CBO will release new 10-year economic projections on 7/2/2020, this PR will be re-updated after this release.

**Comment by chusloj** [comment]
Weights have been recalculated using the newest CBO data. All variables listed under [**Variables in 10-Year CBO Projections**](https://github.com/PSLmodels/taxdata/blob/master/puf_stage1/doc/CBO_Baseline_Updating_Instructions.md#variables-in-10-year-cbo-projections) have been updated with July 2020 data, and unemployment compensation (**UCOMP**) has been updated with March 2020 data.

File: puf_stage1/updatecbo.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import os
 import requests
 import pandas as pd
+import numpy as np
+import xlrd
```
[the remark this answers is not in the document]
Comment (chusloj) [reply puf_stage1/updatecbo.py:5]: Not sure how xlrd ended up there, but I think I used numpy for the `to_numeric` command in lines 97-98.

[the remark this answers is not in the document]
Comment (chusloj) [reply puf_stage1/updatecbo.py:5]: Not sure how xlrd ended up there, but I think I used numpy for the `to_numeric` command in lines 97-98.

**Comment by Peter-Metz** [comment]
@chusloj thanks a lot for the PR and sorry for the slow review. Things look good to me, but I have a couple of minor questions:
- What was the thinking for removing the README?
- Do the [dates](https://github.com/PSLmodels/taxdata/blob/master/puf_stage1/doc/CBO_Baseline_Updating_Instructions.md#cgns) in the docs for CGNS data need to be updated? It looks like the CGNS numbers changed but I'm not sure.

For peace of mind, it might be nice to produce tables that illustrate the aggregate changes in Tax-Calc statistics similar to the ones in [#331 (comment)](https://github.com/PSLmodels/taxdata/pull/331#issuecomment-594674948)

File: puf_stage1/updatecbo.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Peter-Metz) [reply puf_stage1/updatecbo.py:5]: Looks like `to_numeric` is a Pandas [function](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.to_numeric.html)

[the remark this answers is not in the document]
Comment (chusloj) [reply puf_stage1/updatecbo.py:5]: `ubdatecbo.py` runs successfully without `xlrd` and `numpy`. Removing those 2 packages.

**Comment by chusloj** [comment]
> What was the thinking for removing the README?

Not sure how that disappeared. `README.md` from the `master` branch has been restored to the repo.

> Do the dates in the docs for CGNS data need to be updated?

The new values are the exact decimal numbers from the most recent CBO budgetary projections. The previous numbers were just the integer approximations (to one decimal point).

**Comment by chusloj** [comment]
@Peter-Metz Here's the diff tables you suggested above.

Pre-PUF change:

puf_before

Post-PUF change:

puf_after


These tables are for changes in the PUF weights only because the number of records in the new CPS weights file is different from the current CPS weights file in Tax Calculator which was compiled before #314 was merged. I'll hold off on making a CPS comparison until that new CPS weight file pre-CBO update is added to Tax Calculator. @andersonfrailey

**Comment by Peter-Metz** [comment]
Thanks for the tables @chusloj. The magnitude and direction of the changes make sense to me. Think that resolves all of my questions

cc @MattHJensen

**Comment by andersonfrailey** [comment]
This looks good to me, @chusloj. @Peter-Metz good with me merging this and then updating your tax-calc pr?

**Comment by Peter-Metz** [comment]
@andersonfrailey yep, good to merge! We might want to keep [Tax-Calculator#2444](https://github.com/PSLmodels/Tax-Calculator/pull/2444) and the PR for CBO updates separate to be able to separate out how results change for each data modification

**Comment by andersonfrailey** [comment]
@Peter-Metz I think that's a good idea. Anything you want to add/update with this, @chusloj ? I'll merge if you're all good.

**Comment by chusloj** [comment]
@andersonfrailey Ready to merge.

**Pull request merged**